cfgov/v1/jinja2/v1/includes/molecules/tags.html

```diff
@@ -20,30 +20,30 @@
 
    ========================================================================== #}
 
-{% macro render(tags, show_heading=true) %}
-<div class="m-tags">
+{% macro render(tags, show_heading=true, stack_tags=false) %}
+<div>
     {% if show_heading %}
-    <span class="m-tags__heading">
-        Topics:
-    </span>
+    <h4>
+        Topics
+    </h4>
     {% endif %}
-    <ul class="m-tags__list" aria-label="Topics">
+    <ol class="m-tag-group{{ ' m-tag-group--stacked' if stack_tags else '' }}" aria-label="Topics">
```

Comment:
Why use `ol` instead of `ul`? Our topics (and most/all of our "tag" usages in general) are in an arbitrary order.

Reply:
It was because of the alphabetized tag lists, but yes I see it's not always alphabetized and also the order doesn't matter for list of tags meaning, so I agree it should be `ul`. Fixed in https://github.com/cfpb/consumerfinance.gov/pull/8566/commits/5530475c9b0211c017104d4691d26b449c9bf15f